apps/server/src/api-data/excel/excel.service.ts

```diff
@@ -23,32 +24,33 @@ export async function saveExcelFile(filePath: string) {
   if (extname(filePath) != '.xlsx') {
     throw new Error('Wrong file format');
   }
-  excelData = xlsx.parse(filePath, { cellDates: true });
+  excelData = xlsx.readFile(filePath, { cellDates: true, cellFormula: false });
```

Comment:
question: did you try to see what happens if we pass `cellFormula: true`?

Reply:
yes then we get the raw excel formular